src/screens/OtherProfile/index.js

```diff
@@ -387,11 +308,11 @@ const OtherProfile = () => {
     };
 
     const __renderMessageAndFollowButtonGroup = () => {
-      if (blockStatus.blocker) return <></>;
+      if (isBlocking) return <></>;
       return <React.Fragment>{__renderFollowingButton()}</React.Fragment>;
     };
 
-    if (blockStatus.blocked) return <></>;
+    if (isBlocked) return <></>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditions `if (isBlocking)` and `if (isBlocked)` replace the previous `if (blockStatus.blocker)` and `if (blockStatus.blocked)`. Ensure that `isBlocking` and `isBlocked` are equivalent to `blockStatus.blocker` and `blockStatus.blocked` respectively to avoid any behavioral changes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->